test/jdk/sun/net/www/http/HttpClient/MultiThreadTest.java

```diff
@@ -25,6 +25,11 @@
  * @test
  * @bug 4636628
  * @summary HttpURLConnection duplicates HTTP GET requests when used with multiple threads
+ * @run main MultiThreadTest
```

Comment:
do we really need to run this test 5 times?

Reply:
Good point. I will remove them. That was just my trick for increasing the chances of failures while testing the change.